### PR TITLE
Fixed #2419 - Ability to edit and delete a note from Notes in client profile

### LINKED ISSUE #2419: Ability to edit and delete a note from Notes in client profile
For now you can only add a note but after that you can not edit or delete it.
If it happens you want to correct the note you wrote, you will have to write another to correct the previous
notes

It is better to have edit and delete functionality

### PR DESCRIPTION
**Pull request by MYKatz**
Added functionality to edit or delete a previously written note on the 'viewclient' screen.  Deletes/changes notes in both the scope and the backend (thru API). See screenshot:

screencapture-mifos-k4tz-c9users-io-1512702617700

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature `Thanks for the PR, It will be reviewed Shortly!:1eN9Di:cs45fVWUwyukRGPgxBdu_CZWSQs`

**Comment by MYKatz** [comment]
@gkrishnan724 Thank you for the suggestion.  I've updated the commit with the changes you've requested.  Please let me know if I have missed anything.

Thanks!

File: app/views/clients/viewclient.html [hunk 1]
```diff
@@ -1031,8 +1059,11 @@ <h4><strong>{{client.displayName}}</strong></h4>
 					<br/>
 					<div ng-repeat="clientNote in clientNotes | orderBy:predicate:reverse">
 						<blockquote>
-							<p>{{clientNote.note}}</p>
-							{{clientNote.id}}
+							<span class="pull-right noteaddbuttonwrapper">
```
Comment (gkrishnan724) [inline app/views/clients/viewclient.html]: Add `btn-group` in the class, and we should be good to go!

**Comment by MYKatz** [comment]
@gkrishnan724 Done :)

**Pull request merged**